docker-compose.yml

```diff
@@ -2,30 +2,30 @@ version: '3.4'
 services:
   db:
     environment:
-      - MYSQL_USER=foreman
-      - MYSQL_PASSWORD=foreman
-      - MYSQL_DATABASE=foreman
-      - MYSQL_DATADIR_ACTION=upgrade-auto
+      - POSTGRES_USER=foreman
+      - POSTGRES_PASSWORD=foreman
+      - POSTGRES_DATABASE=foreman
+      - PGDATA=/var/lib/postgresql/data/pgdata
```

Comment:
This looks like the standard path, is this here just for better visibility?

Reply:
it actually adds another subdir (pgdata under data) this ensures that the mounted volume is not within the container (see https://hub.docker.com/_/postgres) it follow a similar issue I had on the openshift implementation (at https://github.com/ohadlevy/foreman-kube)